conf/logback.xml

```diff
@@ -26,17 +26,17 @@
 	</appender>
 
 	<logger name="play" level="INFO" />
-	<logger name="application" level="INFO" />
-	<logger name="controllers" level="INFO" />
-	<logger name="models" level="INFO" />
+	<logger name="application" level="DEBUG" />
+	<logger name="controllers" level="DEBUG" />
+	<logger name="models" level="DEBUG" />
 
```

Comment:
Bad idea. We don't want debug mode by default in production. But we can duplicate logback.xml to test/tesources to have debug mode for tests

Reply:
We configure which level should be in application.conf. logback is root settings.